apps/frontend/src/pages/access-management/access-management.tsx

```diff
@@ -62,17 +82,26 @@ const AccessManagement = (): JSX.Element => {
 				<div className={styles["users-table"]}>
 					<Table<UserRow> columns={userColumns} data={userData} />
 					<TablePagination
```

Comment:
What about query parameters for page and pageSize for both tables? they won't work properly with the same name, so we need to rename them

Reply:
This has now been adjusted, it now uses queryParameterPrefix to create the query parameters separetly for both tables
`	const {
		onPageChange: onGroupPageChange,
		onPageSizeChange: onGroupPageSizeChange,
		page: groupPage,
		pageSize: groupPageSize,
	} = usePagination({
		queryParameterPrefix: "group",
		totalItemsCount: groupsTotalCount,
	});`